### PR TITLE
Contributions

### PR DESCRIPTION
**Pull request by cupofcake92**
Hey Jordan, here is the first pass at a master contributors.md file. I've tried to be as general as possible and modeled it a bit on Katia's contributors.md for the eslint repo's, sticking with your preferred coding conventions for pull requests, commit messages, etc. Let me know if you think any other sections should be added back in and I will adjust accordingly (things such as templates, docs, how to request an enhancement, a recognition model, or even who is involved I removed as I wasn't certain if you wanted the information in).

### CODE REVIEW

File: CONTRIBUTIONS.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Contributing 
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to all the repositories managed by Jordan Harband, which are hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```
Comment (ljharb) [inline CONTRIBUTIONS.md]:
```suggestion
The following is a set of guidelines for contributing to all the repositories managed by [@LJHarb](https://github.com/ljharb), which are hosted on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```

File: CONTRIBUTIONS.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+# Table of Contents
```
Comment (ljharb) [inline CONTRIBUTIONS.md]: markdown files on github all have a "table of contents" button in the upper left corner now, so i don't think we need to repeat it here

File: CONTRIBUTIONS.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+## Resolving existing issues 
+
+You can consider helping out with issues already requiring attention - look for a "help wanted" label. 
+
+## Reporting issues 
+
+If you run into problems in the project, you can report them by opening a new issue within the repository. Before filing an issue, please perform a cursory search to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+### How Do I Submit a (Good) Bug Report?
```
Comment (ljharb) [inline CONTRIBUTIONS.md]: let's add something like "if there's an issue template, please always fill it out in its entirety - the template is there for a reason"

File: CONTRIBUTIONS.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+* Install [Node.js](https://nodejs.org/en/), preferably with [nvm](https://github.com/nvm-sh/nvm)
+
+* [Fork the repo and clone your fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+
+* Get all dependencies
+  ```
+  npm install
+  ```
+* Add the upstream source for being able to sync main project changes back into your fork. For example, to configure an upstream remote repository for the `repo-report` fork, run:
```
Comment (ljharb) [inline CONTRIBUTIONS.md]:
```suggestion
* Add the upstream source for being able to sync main project changes back into your fork. For example, to configure an upstream remote repository for a fork of https://github.com/ljharb/repo-report, run:
```

File: CONTRIBUTIONS.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+> **Note:**  Add co-authors to your commit message for commits with multiple authors
+
+```
+Co-authored-by: Name Here <email@here>
+```
+
+
+# Code of Conduct 
+[Code of Conduct](https://github.com/ljharb/.github/blob/main/CODE_OF_CONDUCT.md)
```
Comment (ljharb) [inline CONTRIBUTIONS.md]:
```suggestion
[Code of Conduct](https://github.com/ljharb/.github/blob/HEAD/CODE_OF_CONDUCT.md)
```

File: CONTRIBUTIONS.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+# Where can I ask for help? 
+If you have any questions, please contact <ljharb@gmail.com>.
```
Comment (ljharb) [inline CONTRIBUTIONS.md]:
```suggestion
If you have any questions, please contact [@LJHarb](mailto:ljharb@gmail.com).
```

**Review by ljharb — COMMENTED** [review]
Looks pretty good overall!

**Review by ljharb — APPROVED** [review]

**Comment by ljharb** [comment]
@dani8439 ah one more thing - https://github.com/ljharb/.github/pull/3#discussion_r848951455 ?

**Comment by cupofcake92** [comment]
Ha! Missed that, will remove now.

**Pull request merged**